src/index.js

```diff
@@ -1379,7 +1379,7 @@ export function index(){
         </div>
         <div class="stringPack setting">
             <button id="stringPack" class="button" @click="importStringFile">{{ 'load_string_pack' | label }}</button>
-            <input type="file" class="fileImport" id="stringPackFile" accept=".txt">
+            <input type="file" class="fileImport" id="stringPackFile" accept=".json, text/plain">
```

Comment:
If we're switching to mime types, should we use `application/json` here?
```suggestion
            <input type="file" class="fileImport" id="stringPackFile" accept="text/plain, application/json">
```

Reply:
No wonder I couldn't find a text/json. Yes, this seems better.